### ISSUE TITLE
dadaptadam was wrong

### ISSUE THREAD

**Issue by Aegis-33** [bmaltais/kohya_ss#1464]
When I use Optimizer-dadaptadam, the learning rate becomes a flat horizontal line, which causes the model I train to learn nothing at all
![image](https://github.com/bmaltais/kohya_ss/assets/127716667/265154fe-da31-43dd-ae45-775fb66f2a26)
